src/content/pages/index.html

```diff
@@ -29,7 +29,7 @@ <h1 class="section-title">{{ site.homepage.title }}</h1>
                 <div class="eslint-install-code-wrapper">
                     <div class="eslint-install-code" role="region" aria-labelledby="eslint-installation">
                         <span hidden id="eslint-installation">{{ site.homepage.install.title }}</span>
-                        <button class="eslint-install-code__btn" id="copyBtn" aria-labelledby="copy-button-label" hidden>
```

Comment:
Removed hidden attribute to avoid layout shift.



Before:

https://user-images.githubusercontent.com/30730124/172023126-069a6750-748c-426f-b7a3-c244c2d1e29e.mov

After:


https://user-images.githubusercontent.com/30730124/172023139-ae91ee8c-82f1-48a1-9f4b-eab422a6a89a.mov